resources/views/bread/edit-add.blade.php

```diff
@@ -90,16 +90,22 @@
                                         @endif
                                         <input type="file" name="{{ $row->field }}">
                                     @elseif($row->type == "select_dropdown")
```

Comment:
Yes, will need to make them into their separate view before merging `master` into `v0.11`.
Anyhow, thanks for reminding me.